webxdc.d.ts

```diff
@@ -24,7 +27,10 @@ type ReceivedStatusUpdate<T> = {
   max_serial: number;
   /** optional, short, informational message. */
   info?: string;
-  /** optional, short text, shown beside the webxdc's icon. */
+  /** optional, if the Webxdc creates a document, this is the name of the document;
+   * not set if the Webxdc does not create a document */
+  document?: string;
+  /** optional, short text, shown beside app icon. */
```

Comment:
```suggestion
  /** optional, short text, shown beside the webxdc's icon. */
```

Reply:
i'd prefer this over the other variant.

Reply:
back to app?

Reply:
no :)

i meant that the wording "beside the icon" from the other change is a bit less clear than this one. but both are fine, and repetition is also an issue :) so, everyrhing's fine. sorry for the noise.